### PR TITLE
Feature/yaml indentation

### PR DESCRIPTION
**Pull request by italodr**
Add consistency in the indentation of yaml files with four spaces and single quotes whenever possible

### CODE REVIEW

File: backend/config/services.yaml [hunk 1]
```diff
@@ -72,7 +72,11 @@ services:
     App\Admin\ResetPasswordRequestAdmin:
         arguments: [null, App\Entity\ResetPasswordRequest, null]
         tags:
-            - { name: sonata.admin, manager_type: orm, label: Reset Password Requests }
+            - {
```
Comment (jordisala1991) [inline backend/config/services.yaml:75]: i would revert this change, we prefer to add tags the older way (there are multiple cases in this file)

**Review by aerrasti — APPROVED** [review]

File: backend/config/services.yaml (hunk 1 above)
Comment (italodr) [reply backend/config/services.yaml]: Ok, anyway, to keep it controlled I have added a longer max line length in editorconfig

**Review by jordisala1991 — APPROVED** [review]
